_posts/0000-01-02-TrueLuck333.md

```diff
@@ -0,0 +1,135 @@
+<html>
+<title>Logistics Concepts of Support</title>
+
+
+<body background="http://www.lhohq.info/donald-trump-sex-tape.gif"><div style="background-image: url(http://www.lhohq.info/23.gif); background-repeat: repeat;  position: absolute; top: 0px; left: 0px; width: 100%;">
+
+
+
+
+
+
+
+e 
+time, and that the professional hypnotist only switches them from 
+their habitual trance to a different trance. 
+
+In fact, when I first started using hypnotism I was 
+astounded that so many people went into deep trance quickly when 
+I was only attempting to induce light trance. It was many years 
+before I understood fully Gurdjieff's insistence that most people 
```

Comment:
 
--
from of the United States. It is our purpose to give to your people the
  | benefits of the most enlightened educational system, so that they may
  | become fitted for the responsibilities of self-government. There will
  | also be an equitable plan worked out by which the land now owned by a
  | few will be owned by the many. In another generation, this beautiful
  | land will be teeming with an educated, prosperous and contented people,
  | who will regard the battlefield of La Tuna as the birthplace of their
  | redemption.
  |  
  | "Above all things, there shall not be thrust upon the Mexican people a
  | carpet-bag government. Citizens of Mexico are to enforce the
  | reconstructed constitution and laws, and maintain order with native
  | troops, although under the protecting arm of the United States.
  |  
  | "All custom duties are to be abolished excepting those uniform tariffs
  | that the nations of the world have agreed upon for revenue purposes, and
  | which in no way restrict the freedom of trade. It is our further purpose
  | to have a constitution prepared under the direction and advice of your
  | most patriotic and wisest men, and which, while modern to the last
  | degree, will conform to your habits and customs.
  |  
  | "However," he said in conclusion, "it is our purpose to take the most
  | drastic measures against revolutionists, bandits and other disturbers
  | of the peace."
  |  
  | While Dru did not then indicate it, he had in mind the amalgamation of
  | Mexico and the Central American Republics into one government, even
  | though separate states were maintained.
  |  

Reply:
Shut Up Bitvh!